src/components/view/InfoCard.vue

```diff
@@ -302,10 +302,19 @@
           <a-icon type="rocket" />
           <router-link :to="{ path: '/managementserver/' + resource.managementserverid }">{{ resource.managementserver || resource.managementserverid }}</router-link>
         </div>
-
         <div class="resource-detail-item" v-if="resource.created">
           <a-icon type="calendar" />{{ resource.created }}
         </div>
+        <div class="resource-detail-item" v-if="resource.affinitygroup && resource.affinitygroup.length > 0">
+          <a-icon type="swap" />
+          <span
+            v-for="(group, index) in resource.affinitygroup"
+            :key="group.id"
+          >
+            <router-link :to="{ path: '/affinitygroup/' + group.id }">{{ group.name }}</router-link>
+            <span v-if="index + 1 < resource.affinitygroup.length">, </span>
```

Comment:
I think that's not the smartest way to display the list of affinity groups. Maybe we can change this later. I would prefer to focus on the wizard and adjust the info card later. Or do you have a suggestion, @rhtyd ?

![Bildschirmfoto 2019-12-27 um 14 30 25](https://user-images.githubusercontent.com/13260952/71518912-78da8580-28b5-11ea-86af-60ac8c83dd41.png)